### PR TITLE
Prow pubsub readme: mention project and topic are used for publishing…

### PR DESCRIPTION
**Pull request by chaodaiG**
… statueses

### CODE REVIEW

**Comment by chaodaiG** [comment]
/cc @cjwagner 
cc @2nd-half

File: prow/cmd/sub/README.md [hunk 1]
```diff
@@ -65,8 +65,12 @@ and value ```prow.k8s.io/pubsub.PeriodicProwJobEvent```, and a payload like so:
     "myLabel":"myValue",
   },
   "annotations":{
+    // GCP project where prowjobs statues are published by prow. Must also provide "prow.k8s.io/pubsub.topic" to take effect.
```
Comment (fejta) [inline prow/cmd/sub/README.md:68]: Technically support comments, I'd suggest changing L57 to yaml

**Review by fejta — APPROVED** [review]
/hold

File: prow/cmd/sub/README.md (hunk 1 above)
Comment (chaodaiG) [reply prow/cmd/sub/README.md]: good idea! done

**Review by fejta — APPROVED** [review]

**Pull request merged**